### ISSUE TITLE
Consider publishing xdfile on PyPI

### ISSUE THREAD

**Issue by davide125** [century-arcade/xd#62]
Having the library published on https://pypi.org/ would make it easier to use from other projects. Thanks!

**Comment by saulpw** [comment]
Thanks for the feedback, @davide125!  Are you interested in using the xdfile library on a project yourself?

**Comment by davide125** [comment]
Yes, we were considering using this for [GNOME Crosswords](https://gitlab.gnome.org/jrb/crosswords) to add support for .xd puzzles (https://gitlab.gnome.org/jrb/crosswords/-/issues/112).
